card/spec/lib/card/format/render_spec.rb

```diff
@@ -8,6 +8,7 @@
 
   describe "view cache" do
     before { Cardio.config.view_cache = true }
+    after { Cardio.config.view_cache = false }
```

Comment:
without this, view caching stays on for all tests that follow...  It's good to test caching but confusing to do it arbitrarily.

Reply:
... and insane that this all we have to test the view cache system

Reply:
y, I'll work on that.

Reply:
(as part of the upcoming change when I switch over to the new opt-in caching.)